tests/UI/pages/BO/catalog/categories/index.js

```diff
@@ -10,8 +10,10 @@ class Categories extends BOBasePage {
 
     // Selectors
     this.editHomeCategoryButton = '#main-div .breadcrumb a[href*=\'edit\']';
+
```

Comment:
The class is missing its description and the constructor too

Reply:
Done thx :+1: